sdks/python/apache_beam/testing/benchmarks/inference/pytorch_image_classification_benchmarks.py

```diff
@@ -0,0 +1,63 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# pytype: skip-file
+
+import logging
+
+from apache_beam.examples.inference import pytorch_image_classification
+from apache_beam.testing.load_tests.load_test import LoadTest
+from torchvision import models
+
+_PERF_TEST_MODELS = ['resnet50', 'resnet101', 'resnet152']
+_PRETRAINED_MODEL_MODULE = 'torchvision.models'
+
+
+class PytorchVisionBenchmarkTest(LoadTest):
```

Comment:
Right now, this only runs `pytorch_image_classification`. Are we going to add `pytorch_image_segmentation` to this class as well?

Reply:
Right now, I planned to use one example from vision and one example from NLP. In the future, few more vision tests will be added.

Reply:
Got it, thanks.